models/employees/employee_preferences/preferences.py

```diff
@@ -6,7 +6,7 @@
 
 @dataclass
 class Preferences:
-    days_cannot_work: list[DayOffPreference] = field(default_factory=list)
-    days_prefer_not_to_work: list[DayOffPreference] = field(default_factory=list)
+    days_cannot_work: DayOffPreference = field(default_factory=DayOffPreference)
+    days_prefer_not_to_work: DayOffPreference = field(default_factory=DayOffPreference)
```

Comment:
soon one will be able to prefer a certain day. which is why DayOffPreference is a bad name. its just a day preference. and TBH it should just be a time range and not a day